src/components/diagrams/diagram-pane.jsx

```diff
@@ -234,7 +234,7 @@ const useDisplayView = (studyUuid, currentNode) => {
             } else if (diagramState.svgType === DiagramType.SUBSTATION) {
                 return createSubstationDiagramView(diagramState.id, diagramState.state);
             } else if (diagramState.svgType === DiagramType.NETWORK_AREA_DIAGRAM) {
-                return createNetworkAreaDiagramView(diagramState.ids, diagramState.state, diagramState.depth);
+                return createNetworkAreaDiagramView(diagramState.ids.sort(), diagramState.state, diagramState.depth);
```

Comment:
If we want to open NAD for multiple VLs using a selection map, sorting the IDs can be beneficial. However, in cases where NAD is opened one by one, it may be preferable to maintain the order in which the user requested it.